fixtures/shehds/led-spot-60w.json

```diff
@@ -0,0 +1,430 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "LED Spot 60W",
+  "categories": ["Moving Head"],
+  "meta": {
+    "authors": ["yids"],
+    "createDate": "2022-11-08",
+    "lastModifyDate": "2022-11-08",
+    "importPlugin": {
+      "plugin": "qlcplus_4.12.1",
+      "date": "2022-11-08",
+      "comment": "created by Q Light Controller Plus (version 4.12.3)"
+    }
+  },
+  "links": {
+    "productPage": [
+      "https://www.shehds.com/products/good-technology-led-60w-gobos-moving-head-lighting-for-dmx512-stage-effect-dj-festivals-disco-home-entertainments-in-9-11-channels"
+    ],
+    "manual": [
+      "https://ueeshop.ly200-cdn.com/u_file/UPAG/UPAG904/2005/file/f56ea62fb9.pdf"
+    ],
+    "video": [
+      "https://www.youtube.com/watch?v=2ob8XVgcoRw"
+    ]
+  },
+  "physical": {
+    "dimensions": [19.8, 27.2, 14.4],
+    "weight": 3.2,
+    "power": 60,
+    "bulb": {
+      "type": "LED"
+    },
+    "DMXconnector": "3-pin"
+  },
+  "wheels": {
+    "Color wheel": {
+      "slots": [
+        {
+          "type": "Color",
+          "name": "White"
+        },
+        {
+          "type": "Color",
+          "name": "Red"
+        },
+        {
+          "type": "Color",
+          "name": "Orange"
+        },
+        {
+          "type": "Color",
+          "name": "Lemon Yellow"
+        },
+        {
+          "type": "Color",
+          "name": "Green"
+        },
+        {
+          "type": "Color",
+          "name": "Blue"
+        },
+        {
+          "type": "Color",
+          "name": "Pink"
+        },
+        {
+          "type": "Color",
+          "name": "Sky Blue"
+        }
+      ]
+    },
+    "Gobo wheel": {
+      "slots": [
+        {
+          "type": "Gobo",
+          "name": "Solid"
+        },
+        {
+          "type": "Gobo",
+          "name": "Rose"
+        },
+        {
+          "type": "Gobo",
+          "name": "Radial Dots"
+        },
+        {
+          "type": "Gobo",
+          "name": "Triangle"
+        },
+        {
+          "type": "Gobo",
+          "name": "Hexagon"
+        },
+        {
+          "type": "Gobo",
+          "name": "Swirl"
+        },
+        {
+          "type": "Gobo",
+          "name": "Swirl"
+        },
+        {
+          "type": "Gobo",
+          "name": "Jigsaw"
+        }
+      ]
+    }
+  },
+  "availableChannels": {
+    "Pan": {
+      "fineChannelAliases": ["Pan fine"],
+      "capability": {
+        "type": "Pan",
+        "angleStart": "0%",
+        "angleEnd": "100%",
+        "helpWanted": "Can you provide exact angles?"
+      }
+    },
+    "Tilt": {
+      "fineChannelAliases": ["Tilt fine"],
+      "capability": {
+        "type": "Tilt",
+        "angleStart": "0%",
+        "angleEnd": "100%",
+        "helpWanted": "Can you provide exact angles?"
+      }
+    },
```

Comment:
In the product page, there is an image with the exact angles:

![grafik](https://user-images.githubusercontent.com/202916/201518644-9aca48bc-4dac-42f3-8b34-8dd5b4eb19bf.png)

(even though the gobos don't match the ones from the manual)

```suggestion
    "Pan": {
      "fineChannelAliases": ["Pan fine"],
      "capability": {
        "type": "Pan",
        "angleStart": "0deg",
        "angleEnd": "540deg"
      }
    },
    "Tilt": {
      "fineChannelAliases": ["Tilt fine"],
      "capability": {
        "type": "Tilt",
        "angleStart": "0deg",
        "angleEnd": "270deg"
      }
    },
```